vue.config.js

```diff
@@ -1,103 +1,65 @@
 module.exports = {
   chainWebpack: (config) => {
     config.module
-      .rule("yaml")
+      .rule('yaml')
       .test(/\.ya?ml$/)
-      .use("raw-loader")
-      .loader("raw-loader")
+      .use('raw-loader')
+      .loader('raw-loader')
       .end();
   },
-  publicPath: "",
+  publicPath: '',
   pwa: {
-    manifestPath: "assets/manifest.json",
+    manifestPath: 'assets/manifest.json',
```

Comment:
:thinking: I think we could actually remove all the pwa configuration except from the manifestPath (used to inject the path in the header). 
The goal is to use the static manifest in the asset directory so anybody can customise the app name & other thing easily. What do you think?

Reply:
[@vue/cli-plugin-pwa](https://cli.vuejs.org/core-plugins/pwa.html#configuration) looks like it generates the manifest file unless you put an absolute path in there. I'm not 100% sure, but its possible the JSON file isn't even used right now.